embed_html_images.py

```diff
@@ -44,9 +54,10 @@ def main():
                                                  file to embed image',
                                      formatter_class=argparse.ArgumentDefaultsHelpFormatter)
     parser.add_argument('--file_path', required=True, help='Path of html file to process')
+    parser.add_argument('--out_path', required=False, default=None, help='Path of html file to process')
```

Comment:
Should the help text for --out_path be for the output file instead?

Reply:
yes